docs/digger/installation/using-http-proxy.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[using-http-proxy]]
+== Using HTTP/HTTPS Proxy with Aerogear Digger (optional)
+
+WARNING: HTTP Proxies that require authentication are not supported.
+Please read the link:known_issues[Known Issues] for more details.
+
+You can install AeroGear Digger behind proxy by modifying the variables in Ansible playbook.
+
+The installer will automatically configure the following components to use the proxy (if the component requires an outbound Internet access):
+
+* The Jenkins server
+* The Android SDK pod used to download Android SDKs
+* Container/node that will run the build jobs, by adding the environment variables such as `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file. 
+https://github.com/aerogear/digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file].
```

Comment:
not sure if we need to link this here. We talk about this in the section `Running the Digger Installer`

Reply:
resolved